website/client/src/components/groups/questDetailModal.vue

```diff
@@ -0,0 +1,450 @@
+          <select-translated-array
+            :right="true"
+            :items="['quantity', 'AZ']"
+            :value="sortBy"
+            class="inline"
+            :inline-dropdown="false"
+            @select="sortBy = $event"
+          />
+        </div>
+      </div>
+
+      <div class="quest-items">
+        <!-- eslint-disable vue/no-use-v-if-with-v-for -->
+        <div
+          v-for="item in questsInfoList"
+          :key="item.key"
+          class="quest-col"
+          @click="selectQuest(item)"
+        >
+          <item
+            :key="item.key"
+            :item="item"
+            :item-content-class="item.class"
+          >
+            <countBadge
+              slot="itemBadge"
+              :show="item.amount !== 1"
+              :count="item.amount"
+            />
+            <template
+              slot="popoverContent"
+              slot-scope="context"
+            >
+              <div
+                class="questPopover"
+              >
+                <h4 class="popover-content-title">
+                  {{ context.item.text }}
+                </h4>
+                <questInfo :quest="context.item" />
+              </div>
+            </template>
+          </item>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-10 offset-1 text-center">
+          <span
+            v-once
+            class="no-quest-to-start"
+          >
+            <b>{{ $t('noQuestToStartTitle') }}</b> <br>
+            <span v-html="$t('noQuestToStart', { questShop: '/shops/quests' })"></span>
+          </span>
+        </div>
+      </div>
+    </div>
+    <div v-else>
+      <div v-if="questData" class="quest-combined-content">
+        <questDialogContent
+          :item="questData"
+          :group="group"
+          class="quest-detail"
+        />
+        <quest-rewards :quest="questData" class="mt-4" />
+      </div>
+      <div
+        v-if="!groupHasQuest"
+        class="text-center"
+      >
+        <button
+          class="btn btn-primary mt-0 invite-btn"
+          :disabled="!Boolean(selectedQuest) || loading"
+          @click="questInit()"
+        >
+          {{ $t('inviteToPartyOrQuest') }}
+        </button>
+      </div>
+      <div v-if="fromSelectionDialog"
+           class="text-center back-to-selection"
+           @click="goBackToQuestSelection()">
+        <span v-once
+             class="svg-icon color"
+             v-html="icons.navigationBack">
+        </span>
+
+        <span>
+          {{ $t('backToSelection') }}
+        </span>
+      </div>
+      <div
+        v-if="groupHasQuest && canEditQuest"
+        class="text-center actions"
+      >
+        <div>
+          <button
+            v-if="!onActiveQuest"
+            v-once
+            class="btn btn-success mb-3"
+            @click="questConfirm()"
+          >
+            {{ $t('startQuest') }}
+          </button>
+          <!-- @TODO don't allow the party leader to
```

Comment:
If these aren't addressed within the PR, we should probably write them up as tickets--we don't have a good track record of revisiting our `TODO`s

Reply:
extracted as an Issue: https://github.com/HabitRPG/habitica/issues/13219